### ISSUE TITLE
like picpick

### ISSUE THREAD

**Issue by skypce** [martyr-deepin/deepin-screenshot#5]
Will be great if you can do a tool like picpick for deepin. I love picpick but emulated with wine have some issues yet. Thank you

**Comment by pangpangpang3** [comment]
Yes, it's a wonderful tool, and deepin-screenshot will continue to be optimized.
